features/load-rest.feature

```diff
@@ -2,7 +2,8 @@ Feature: Manage WordPress through endpoints locally
 
   Background:
     Given a WP install
-    And I run `wp plugin install rest-api --activate`
+	When I run `wp core version`
+	Then STDOUT should be a version string >= 4.7
```

Comment:
> I see no issue all tests pass. Perhaps this is a text/tab oddity

Behat `.feature` files require spaces, not tabs. We don't have a linter in place right now.